data/projects/g/galxe.yaml

```diff
@@ -1,4 +1,3 @@
-version: 5
```

Comment:
@mseidlx I think this is preventing the CI validating from passing. We use version numbers to handle schema migrations FYI